packages/forms-web-app/src/views/appellant-submission/site-ownership-certb.njk

```diff
@@ -38,13 +38,13 @@
             </div>
             <div class="govuk-radios govuk-!-padding-bottom-6">
               <div class="govuk-radios__item">
-                <input class="govuk-radios__input" id="have-other-owners-been-told-yes" name="have-other-owners-been-told" type="radio" value="yes" data-cy="answer-yes" {% if haveOtherOwnersBeenTold %}checked{% endif %}>
+                <input class="govuk-radios__input" id="have-other-owners-been-told" name="have-other-owners-been-told" type="radio" value="yes" data-cy="answer-yes" {% if haveOtherOwnersBeenTold %}checked{% endif %}>
                 <label class="govuk-label govuk-radios__label" for="have-other-owners-been-told-yes">
                   Yes, I have already told the other owners
                 </label>
               </div>
               <div class="govuk-radios__item">
-                <input class="govuk-radios__input" id="have-other-owners-been-told-no" name="have-other-owners-been-told" type="radio" value="no" data-cy="answer-no" {% if haveOtherOwnersBeenTold === false %}checked{% endif %}>
+                <input class="govuk-radios__input" id="have-other-owners-been-told-2" name="have-other-owners-been-told" type="radio" value="no" data-cy="answer-no" {% if haveOtherOwnersBeenTold === false %}checked{% endif %}>
```

Comment:
Really!

Reply:
Yes really. If we were to use the nunjucks component, the ID would be set to the plain id, then suffix 2, 3, etc. This is only the ID, the name and value are unaffected.